lib/rules/function-calc-no-unspaced-operator/index.mjs

```diff
@@ -222,46 +135,46 @@ const rule = (primary, _secondaryOptions, context) => {
 					const token = operation.firstOperand.value;
 
 					if (isTokenDimension(token)) {
-						const { unit } = token[4];
-						const operatorChar = unit.slice(-1);
+						const [, , , endPos, { unit }] = token;
```

Comment:
question
minor

Is there a way to keep the `return` after `complain`?

```js
/** @param {'unit'|'value'} property */
const bar = (property) => {
	const [, , , endPos, { [property]: operator }] = token;
	const operatorChar = operator.at(-1);

	if (operatorChar && OPERATOR_REGEX.test(operatorChar)) {
		operation.completeMissingOperator(operatorChar, endPos, 'append');

		complain(
			messages.expectedBefore(operation.operatorChar),
			decl,
			valueIndex + operation.operatorCharPosition,
			operation.operatorChar,
			() => {
				operation.insertOperatorAfterFirstOperand(node);
				mutateUnit(token, operator.slice(0, -1));
				fixDeclarationValue();
			},
		);
	}
};
```

Reply:
Possible. How about the new commit 04df33fdf?